packages/input_schema/src/schema.json

```diff
@@ -79,23 +79,61 @@
         "stringProperty": {
             "title": "String property",
             "type": "object",
-            "additionalProperties": false,
+            "additionalProperties": true,
             "properties": {
                 "type": { "enum": ["string"] },
                 "title": { "type": "string" },
                 "description": { "type": "string" },
-                "default": { "type": "string" },
-                "prefill": { "type": "string" },
                 "example": { "type": "string" },
-                "pattern": { "type": "string" },
                 "nullable": { "type": "boolean" },
-                "minLength": { "type": "integer" },
-                "maxLength": { "type": "integer" },
                 "editor": { "enum": ["javascript", "python", "textfield", "textarea", "hidden"] },
+                "isSecret": { "type": "boolean" },
                 "sectionCaption": { "type": "string" },
                 "sectionDescription": { "type": "string" }
```

Comment:
Maybe you can delete everything but `isSecret`? It will get overriden anyway in the `then` and `else` directives, and if you remove it, it will make the returned errors a bit more readable.

Reply:
I remove some of them, but I need to keep these because we used them in some of our custom validation methods or in error generation and for checking required fields.

Reply:
Ah, right, I was checking it just with AJV, not with our custom validations.